tools/delete_candidate_info.php

```diff
@@ -0,0 +1,431 @@
+        echo "Dropping all DB entries for candidate DCCID: " . $DCCID . "And PSCID:" .
+            $PSCID . "\n";
+
+        //delete the sessions
+        $DB->delete("session", array("CandID" => $DCCID));
+
+        //delete each instrument table entry
+        foreach ($instruments as $instrument) {
+
+            //delete the entry from the instrument table
+            $DB->delete(
+                $instrument['Test_name'], array("CommentID" => $instrument['CommentID'])
+            );
+
+            //delete from flag
+            $DB->delete("flag", array("ID" => $instrument['ID']));
+
+            //delete from conflicts_resolved
+            $DB->delete(
+                "conflicts_resolved", array("CommentId1" => $instrument['CommentID'])
+            );
+            $DB->delete(
+                "conflicts_resolved", array("CommentId2" => $instrument['CommentID'])
+            );
+            //delete from conflicts_unresolved
+            $DB->delete(
+                "conflicts_unresolved", array("CommentId1" => $instrument['CommentID'])
+            );
+            $DB->delete(
+                "conflicts_unresolved", array("CommentId2" => $instrument['CommentID'])
+            );
+
+            //delete from final_radiological_review
+            $DB->delete(
+                "final_radiological_review", array("CommentID" => $instrument['CommentID'])
+            );
+        }
+
+        //Delete from the feedback related tables
+        foreach ($Feedbackids as $Feedbackid) {
+            $DB->delete(
+                "feedback_bvl_entry", array('FeedbackID' => $Feedbackid['FeedbackID'])
+            );
+        }
+        $DB->delete("feedback_bvl_thread", array('CandID' => $DCCID));
+
+        //delete from the participant_status table
+        $DB->delete("participant_status", array("CandID" => $DCCID));
+
+        //delete from the participant_status_history table
+        $DB->delete("participant_status_history", array("CandID" => $DCCID));
+
+        //delete from parameter_candidate
+        $DB->delete("parameter_candidate", array("CandID" => $DCCID));
+
+        //delete from candidate
+        $DB->delete("candidate", array("CandID" => $DCCID));
+    }
+}
+
+function deleteTimepoint($sessionID, $confirm) {
+    $DB =& Database::singleton();
+
+    $instruments = $DB->pselect('SELECT Test_name, CommentID FROM flag WHERE SessionID=:sid', array('sid' => $sessionID));
+
+    // Print each instrument instance
+    foreach ($instruments as $instrument) {
+        $result = $DB->pselect(
+            'SELECT * FROM ' . $DB->escape($instrument['Test_name']) . ' WHERE CommentID=:cid',
+            array('cid' => $instrument['CommentID'])
+        );
+        print_r($result);
+
+        // Print from conflicts
+        $result = $DB->pselect(
+            'SELECT * FROM conflicts_unresolved WHERE CommentId1=:cid OR CommentId2=:cid',
+            array('cid' => $instrument['CommentID'])
+        );
+        print_r($result);
```

Comment:
Same here: 
print "conflicts_unresolved data: "

and same for the other print_r below.